src/guide/components/navigation/Navigation.jsx

```diff
@@ -15,80 +15,73 @@ export default class Navigation extends Component {
 
   constructor(props) {
     super(props);
-
-    this.state = {
-      isMenuOpen: false,
-    };
-
-    this.onClickNavButton = this.onClickNavButton.bind(this);
-    this.onToggleNav = this.onToggleNav.bind(this);
-  }
-
-  onToggleNav() {
-    this.setState({
-      isMenuOpen: !this.state.isMenuOpen,
-    });
-  }
-
-  onClickNavButton() {
-    this.setState({
-      isMenuOpen: false,
-    });
   }
 
   render() {
-    const addRoutesToNav = function addRoutesToNav(name, routeItems, navItems) {
+    const addRoutesToNav = (name, routeItems, navItems) => {
       navItems.push(
         <NavTitle key={name}>{name}</NavTitle>
       );
       for (let i = 0; i < routeItems.length; i++) {
         const route = routeItems[i];
         navItems.push(
-          <NavButton
+          <NavItem
             href={route.href}
             path={route.path}
-            onClick={this.onClickNavButton}
+            onClick={this.props.onClickNavItem}
             key={`${name}${i}`}
           >
             {route.name}
-          </NavButton>
+          </NavItem>
         );
       }
-    }.bind(this);
+    };
 
-    const navItems = [];
+    const componentNavItems = [];
+    addRoutesToNav('Components', this.props.componentRoutes, componentNavItems);
```

Comment:
I see. But I think in general it's a good idea to use multiple lines if there are multiple arguments. Would you agree?